### PR TITLE
Integrated Mayhem

### PR DESCRIPTION
**Pull request by bmwtsn098**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Build Stage:
```
Comment (xansec) [inline Dockerfile:1]: Please add a package stage to reduce image size.

File: Mayhemfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Namespaced project name that the target belongs to
```
Comment (xansec) [inline Mayhemfile:1]: Please use generic project name, and omit the duration and image.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

**Comment by bmwtsn098** [comment]
I've made the requested changes. Let me know if everything looks good. Thanks

**Review by xansec — APPROVED** [review]
LGTM!

**Pull request merged**